modules/virt-about-conditions-and-events.adoc

```diff
@@ -7,6 +7,7 @@
 
 Diagnose DataVolume issues by examining the output of the `Conditions` and `Events` sections
 generated by the command:
+[source, terminal]
```

Comment:
For all instances of `[source, terminal]`, please take out the space after the comma so that it reads `[source,terminal]`.

Reply:
Done.